src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/Phase2Incentive.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.ValueTypes;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ValueObjects
+{
+    public class Phase2Incentive : Incentive
+    {
+        public Phase2Incentive(
+            DateTime dateOfBirth,
+            DateTime startDate,
+            IEnumerable<PaymentProfile> paymentProfiles,
+            IReadOnlyCollection<BreakInLearning> breakInLearningDayCount,
+            DateTime submissionDate) : base(dateOfBirth, startDate, paymentProfiles, breakInLearningDayCount, submissionDate)
+        {
+        }
+
+        public static DateTime EligibilityStartDate = new DateTime(2021, 4, 1);
+        public static DateTime EligibilityEndDate = new DateTime(2021, 11, 30);
+        public override bool IsEligible => StartDate >= EligibilityStartDate && StartDate <= EligibilityEndDate;
+        protected override int DelayPeriod => 21;
```

Comment:
Should this be pulled from config? AC1 describes a configurable delay period. Maybe not for live running, but could aid with testing?

Reply:
As discussed, that acceptance criteria offers limited value as the delay period only applies when earnings calculations are performed.